RabbitMQ.pm

```diff
@@ -81,26 +83,30 @@ will exit with error code 141
 
 =head2 disconnect()
 
-Causes the connection to RabbitMQ to be torn down.
+Disconnect from the RabbitMQ server.
 
 =head2 get_server_properties()
 
-Get a hashref of server properties (these may vary, you should C<Data::Dumper> to inspect). They will be provided by the RabbitMQ server to which you are connected.
+Get a reference to hash of server properties (these may vary, you should use C<Data::Dumper> to inspect). Properties will be provided for the RabbitMQ server to which you are connected.
```

Comment:
Yeah, so, folks who program Perl know exactly what you mean by `hashref`, and they will also know what you mean by _reference to hash_. However, consider the following two scenarios:

A:

```perl
my $foo = { a => 1 };
bar( $foo );
```

and B:

```perl
my %foo = ( a => 1 );
bar( \%foo );
```

In _A_, you never see the hash. For me, that is a scalar, but it is also a hashref. In _B_, that's a reference to a hash because I have a hash and I'm passing a reference. Does that make sense?

Then there's the linguistics of it, if you can say the same thing two different ways, prefer the shorter way of saying it.